Source/WPEFramework/PluginServer.h

```diff
@@ -1394,1571 +1416,1592 @@ namespace PluginHost {
-                private:
-                    Override(const Override&) = delete;
-                    Override& operator=(const Override&) = delete;
+                    Plugin& operator=(Plugin const& other) = delete;
 
                 public:
-                    class Plugin : public Core::JSON::Container {
-                    private:
-                        Plugin& operator=(Plugin const& other) = delete;
-
-                    public:
-                        Plugin()
-                            : Core::JSON::Container()
-                            , AutoStart()
-                            , Configuration(_T("{}"), false)
-                        {
-                            Add(_T("autostart"), &AutoStart);
-                            Add(_T("configuration"), &Configuration);
-                        }
-                        Plugin(const string& config, const bool autoStart)
-                            : Core::JSON::Container()
-                            , AutoStart(autoStart)
-                            , Configuration(config, false)
-                        {
-                            Add(_T("autostart"), &AutoStart);
-                            Add(_T("configuration"), &Configuration);
-                        }
-                        Plugin(Plugin const& copy)
-                            : Core::JSON::Container()
-                            , AutoStart(copy.AutoStart)
-                            , Configuration(copy.Configuration)
-                        {
-                            Add(_T("autostart"), &AutoStart);
-                            Add(_T("configuration"), &Configuration);
-                        }
+                    Plugin()
+                        : Core::JSON::Container()
+                        , AutoStart()
+                        , Configuration(_T("{}"), false)
+                    {
+                        Add(_T("autostart"), &AutoStart);
+                        Add(_T("configuration"), &Configuration);
+                    }
+                    Plugin(const string& config, const bool autoStart)
+                        : Core::JSON::Container()
+                        , AutoStart(autoStart)
+                        , Configuration(config, false)
+                    {
+                        Add(_T("autostart"), &AutoStart);
+                        Add(_T("configuration"), &Configuration);
+                    }
+                    Plugin(Plugin const& copy)
+                        : Core::JSON::Container()
+                        , AutoStart(copy.AutoStart)
+                        , Configuration(copy.Configuration)
+                    {
+                        Add(_T("autostart"), &AutoStart);
+                        Add(_T("configuration"), &Configuration);
+                    }
 
-                        virtual ~Plugin()
-                        {
-                        }
+                    virtual ~Plugin()
```

Comment:
override?

Reply:
This is code writtten in the time that the MIPS compiler was not supporting std11 (and thus the delete keyword).
We wgreed that we would not actively start updating all the code, but only if we tocuh it. I do not think this code was tocuched so hence it is till prior to the std11 standard (override keyword was not supported on these compilers.... yet)